UnityProject/Assets/Scripts/HealthV2/Living/Cyborg/BatteryPack.cs

```diff
@@ -3,12 +3,58 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class BatteryPack : MonoBehaviour
+public class BatteryPack : MonoBehaviour, IChargeable
 {
 	public List<Battery> Cells = new List<Battery>();
 
 	public ItemStorage Storage;
 
+	public bool FullyCharged()
```

Comment:
```suggestion
	public bool IsFullyCharged()
```
this could also be a property instead. Semantically, a method is an action the object can take, this doesn't do any action.